CHANGELOG.md

```diff
@@ -4,6 +4,25 @@
 
 - "You miss 100 percent of the chances you don't take. — Wayne Gretzky" — Michael Scott
 
+## 7.58.0
+
+- feat(core): Support `AggregateErrors` in `LinkedErrors` integration (#8463)
+- feat(core): Add helpers to get module metadata from injected code (#8438)
+- feat(node): Populate propagation context using env variables (#8422)
+- feat(replay): Add a new option `networkDetailDenyUrls` (#8439)
+- feat(tracing): Add `PropagationContext` to scope  (#8421)
+- feat(tracing): Add tracing without performance to browser and client Sveltekit (#8458)
```

Comment:
Hmm this already is a behaviour change, right? I wonder if we should wait on the remaining TwP PRs and add a paragraph by then 🤔

Reply:
We can ship browser Tracing without performance without needing to have node support.

```
This release adds support for [distributed tracing](https://docs.sentry.io/product/sentry-basics/tracing/distributed-tracing/) without the need for a span or transaction to exist in our Browser (and related framework) SDKs.

If you've configured the `BrowserTracing` integration, we now attach the `sentry-trace` and `baggage` headers even when transactions/spans are not being sampled.
```